cloud/disk_manager/internal/pkg/dataplane/url/common/http_client.go

```diff
@@ -112,6 +112,35 @@ func (l *loggerWithURLReplaced) Error(msg string, keysAndValues ...interface{})
 
 ////////////////////////////////////////////////////////////////////////////////
 
+func checkFor4xxOr5xxStatusCode(statusCode int) error {
```

Comment:
maybe instead of `checkFor4xxOr5xxStatusCode` you can name it `checkHttpStatus` or `checkHttpErrorStatus`